torch/csrc/functorch/CompileCache.cpp

```diff
@@ -185,7 +186,7 @@ struct CompileCache {
     for (int i = numTensorArgs; i < PyTuple_Size(args); i++) {
       PyObject *arg = PyTuple_GET_ITEM(args, i);
       assert(PyLong_Check(arg));
-      cacheKey.push_back(PyLong_AsLong(arg));
+      cacheKey.push_back(THPUtils_unpackLong(arg));
```

Comment:
`PyLong_AsLong` is dangerous on windows (the linter tells me) so I changed it to `THPUtils_unpackLong`